### PR TITLE
Updated the responsiveness of the website

### LINKED ISSUE #192: Responsiveness and design of the website
I think the divs in the know about portion when seen in smaller device will look better if they are placed center.
I also want to work on the design of the website.
Please assign me for this issue under gssoc'21.

### PR DESCRIPTION
**Pull request by Avishake007**
# Description

Here I have updated the responsiveness the website
Fixed #192 

## Type of change

Please check options that are relevant to your PR.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have pulled the latest changes in the repository
- [ ] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by vaishali614** [comment]
@Avishake007 The Know About cards are in center in mobile screen but the changes you made shifted it to left in desktop screen. Please check that. Also, you've changed styles of a few sections. Please undo those changes.

File: src/CSS/KnowAboutCard.module.css [hunk 1]
```diff
@@ -8,14 +8,14 @@
 .Card {
   width: 200px;
   border-radius: 2%;
-  box-shadow: 0px 3px 4px 0px #ccc;
+  /* box-shadow: 0px 3px 4px 0px #ccc; */
```
Comment (vaishali614) [inline src/CSS/KnowAboutCard.module.css]: Undo this change

File: src/CSS/KnowAboutCard.module.css [hunk 2]
```diff
@@ -8,14 +8,14 @@
...
   /* margin-left: 50%; */
   transition: all 0.6s;
 }
 
 .Card:hover {
   z-index: 99;
-  box-shadow: 2px 2px 30px #b9b9b9;
+  box-shadow: 0px 3px 4px 0px #ccc;
```
Comment (vaishali614) [inline src/CSS/KnowAboutCard.module.css]: Undo this change

File: src/Components/HomePage/Sections/KnowAbout.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ class KnowAbout extends Component {
             <span className={styles.About}> About </span>
           </h2>
           <Row
-            className="justify-content-md-center"
+            className={styles.justify_content}
```
Comment (vaishali614) [inline src/Components/HomePage/Sections/KnowAbout.js]: This change has centered cards in mobile view but left aligned them in desktop view. Please check.

**Review by vaishali614 — CHANGES_REQUESTED** [review]

**Review by vaishali614 — APPROVED** [review]

**Comment by vaishali614** [comment]
@Avishake007 Changes look good. Please take the latest pull and update your PR, so that I can merge it.

**Comment by vaishali614** [comment]
> @Avishake007 Changes look good. Please take the latest pull and update your PR, so that I can merge it.

Reminder

**Comment by Avishake007** [comment]
@vaishali614 Could you please explain me in details  what I have to do?

**Comment by vaishali614** [comment]
> @vaishali614 Could you please explain me in details what I have to do?

You need to take the latest pull from upstream by running the following commands:
$ git pull upstream develop
$ git push origin develop

**Comment by Avishake007** [comment]
@vaishali614  Its done

**Comment by vaishali614** [comment]
> @vaishali614 Its done

Your fork is still 26 commits behind. Please check again.

**Pull request merged**